src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/logger.html.twig

```diff
@@ -33,103 +33,136 @@
+            <div class="tab">
+                <h3 class="tab-title">Info. &amp; Errors <span class="badge">{{ info_and_error_logs|length }}</span></h3>
+
+                <div class="tab-content">
+                    {% if info_and_error_logs is empty %}
+                        <div class="empty">
+                            <p>There are no log messages of this level.</p>
+                        </div>
+                    {% else %}
+                        {{ helper.render_table(info_and_error_logs, true) }}
+                    {% endif %}
+                </div>
+            </div>
+
+            <div class="tab">
+                <h3 class="tab-title">Deprecations <span class="badge">{{ deprecation_logs|length }}</span></h3>
+
+                <div class="tab-content">
+                    {% if deprecation_logs is empty %}
+                        <div class="empty">
+                            <p>There are no log messages about deprecated features.</p>
+                        </div>
+                    {% else %}
+                        {{ helper.render_table(deprecation_logs, false, true) }}
+                    {% endif %}
+                </div>
+            </div>
+
+            <div class="tab">
+                <h3 class="tab-title">Debug <span class="badge">{{ debug_logs|length }}</span></h3>
+
+                <div class="tab-content">
+                    {% if debug_logs is empty %}
+                        <div class="empty">
+                            <p>There are no log messages of this level.</p>
+                        </div>
+                    {% else %}
+                        {{ helper.render_table(debug_logs) }}
+                    {% endif %}
+                </div>
+            </div>
+        </div>
+
     {% endif %}
 {% endblock %}
 
+{% macro render_table(logs, show_level = false, is_deprecation = false) %}
+    {% import _self as helper %}
+
+    <table class="auto-layout logs">
+        <thead>
+            <tr>
+                <th>{{ show_level ? 'Level' : 'Time' }}</th>
+                <th>Channel</th>
+                <th>Message</th>
+            </tr>
+        </thead>
+
+        <tbody>
+            {% for log in logs %}
+                {% set css_class = is_deprecation ? ''
+                    : log.priorityName in ['CRITICAL', 'ERROR', 'ALERT', 'EMERGENCY'] ? 'status-error'
+                    : log.priorityName in ['NOTICE', 'WARNING'] ? 'status-warning'
+                %}
+                <tr class="{{ css_class }}">
+                    <td class="font-normal text-small">
+                        {% if show_level %}
+                            <span class="colored text-bold nowrap">{{ log.priorityName }}</span>
+                        {% endif %}
+                        <span class="text-muted nowrap newline">{{ log.timestamp|date('H:i:s') }}</span>
+                    </td>
+
+                    <td class="font-normal text-small text-bold nowrap">{{ log.channel }}</td>
```

Comment:
``log.channel`` may be undefined if you use another PSR-3 logger than Monolog (it is a MonologBundle concept). Please keep the profiler panel compatible with such case (it is in 2.7)